app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -9,28 +9,27 @@
     Your training will now start in <%= (@application_choice.course_option.course.start_date + 1.year).to_s(:month_and_year) %>.
   </p>
 <% elsif @application_choice.pending_conditions? || @application_choice.recruited? || @application_choice.offer? %>
+  <%= govuk_details(
+    summary_text: "What to do if you’re unable to start training in #{@application_choice.course_option.course.start_date.to_s(:month_and_year)}",
+    classes: 'govuk-!-margin-bottom-0 govuk-!-margin-top-2',
+  ) do %>
+    <p>
+      Some providers allow you to defer your offer. This means that you could start your course a year later.
+    </p>
 
-    <%= govuk_details(
-      summary: "What to do if you’re unable to start training in #{@application_choice.course_option.course.start_date.to_s(:month_and_year)}",
-      classes: 'govuk-!-margin-bottom-0 govuk-!-margin-top-2',
-    ) do %>
-      <p>
-        Some providers allow you to defer your offer. This means that you could start your course a year later.
-      </p>
```

Comment:
Can we move the formatting improvements to a new PR, so that it's clear what needs to change as part of the new govuk components gem.

Reply:
Yeah, I did feel a bit uncomfortable when I saw that diff. Fixed – much easier to see what’s changed now. Thanks!